nightly_tests/run.sh

```diff
@@ -227,10 +229,13 @@ if [[ "$RUN_TESTS" == "true" ]]; then
   mv nightly_output_$TODAYS_DATE.txt ${LOG_DIR}
   mv selenium_unity_images/* ${LOG_DIR}
 
+  #Delete logs older then 2 weeks
+  bash delete_old_logs.sh
```

Comment:
Only run this line if "$RUN_TESTS" == "true" .   For normal users of this script (non CI use case), we don't want to incur this call/cost.